### ISSUE TITLE
Repo level IntelliJ Platform settings via `.ijwb/.idea`

### ISSUE THREAD

**Issue by bric3** [bazelbuild/intellij#6787]
### Description of the feature request:

Cannot commit `.idea` settings files within `.ijwb/.idea`, because the `.ijwb` "already exists".

### What underlying problem are you trying to solve with this feature?

The IDEA project folder is generated in `.ijwb/.idea` of the repository root. In regular non-bazel project, it is possible to commit some files of the `.idea` folder to help configure IDEA.
E.g.

* Silly things like the `.idea/icon.svg`
* Run configurations `.idea/runConfigurations/*.xml`
* (Search) Scopes `.idea/scopes/`
* External dependencies (like plugins) `.idea/externalDependencies.xml`

I was hoping that it possible to commit such files `.ijwb/.idea/...`, but it doesn't work because the `.ijwb` folder has to be re-created on each import. Otherwise we see the following error.

<img width="389" alt="image" src="https://github.com/user-attachments/assets/d91d193a-496a-42c5-9086-da8aa0d626a7">

Related issues:

* #434 
* #6454
* #1128

### What operating system, Intellij IDE and programming languages are you using? Please provide specific versions.

macOs, GoLand & Intellij IDEA 2024.2.2, Bzel for Intellij 2024.08.27.0.1-api-version-242

### Have you found anything relevant by searching the web?

Nope

**Comment by tpasternak** [comment]
Duplicate of #3326

**Comment by tpasternak** [comment]
Hey @bric3 . I don't have capacity to work on it, but could experiment with checking some files in `.ijwb/.idea` and then using "File -> open -> .jiwb directory" instead of "import bazel project"

**Comment by bric3** [comment]
> but could experiment with checking some files in `.ijwb/.idea` and then using "File -> open -> .jiwb directory" instead of "import bazel project"

Actually, this doesn't work well, in the sense that the monorepo I'm working on has multiple "views" consisting each of a `.bazelproject`, there's about 100 in this repo. That's why the devx team suggest to _import the project_.

That says thank you for the suggestion and the feedback!

**Comment by tpasternak** [comment]
So just in case you missed that - it is possible to setup run configurations xmls via the project files https://github.com/bazelbuild/intellij/blob/master/examples/java/greetings_project/run_configurations/fragment.bazelproject

Maybe we could think of extending that

**Comment by bric3** [comment]
By the way do you think it is possible to contribute this feature (i.e.  `import_idea_scope`, `import_external_dependencies`, `import_project_icon`)? Or maybe a more generic form because other files I don't know about could be committed in the repo, or because some file names could change as IJ release go on. 

Note, I believe `import_run_configurations` is only processed by this plugin. But is it ?

**Comment by tpasternak** [comment]
Yes, of course, we would be happy to have this feature. My concern is that it might not be that easy, the `import_run_configuration` sections required some specific code here. https://github.com/bazelbuild/intellij/blob/master/base/src/com/google/idea/blaze/base/run/BlazeRunConfigurationSyncListener.java

I'm not sure if it would be possible to handle the generic case


cc @brian-mcnamara you might be interested in this
